app/controllers/trucks_controller.rb

```diff
@@ -1,5 +1,7 @@
 class TrucksController < ApplicationController
-  def index
```

Comment:
we should be leaving our actions in even if they are blank
its confusing to look at a controller that can render index but doesnt have it defined.
Also i had no idea if you have a helper method and no defined action it will still render